### PR TITLE
SDK-1321 add readme file with instructions to run the examples

### PR DESCRIPTION
**Pull request by dtayeh**
### :pencil: Description
- 

### :link: Related Issues
-

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- `log4j-slf4j2-impl`: Apache Log4j SLF4J Binding.
+
+## Requirements
+
+- Java 1.8 or higher
+- Maven
+
+## License
+
+This project is licensed under the terms of the MIT license.
```
Comment (mohnoor94) [inline examples/README.md:43]: We use [Apache License 2.0](https://github.com/ExpediaGroup/rapid-java-sdk/blob/main/LICENSE).

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+1. Clone the repository.
+2. Navigate to the project directory `examples`.
+3. Acquire your own API key and secret from Expedia Group. Replace the placeholders `your_api_key` and `your_api_secret` in the `RapidService` class with your actual API key and secret.
+4. Run `mvn clean install` to build the project and install the dependencies including the RAPID SDK.
+
+## Running the Examples
+
+To run the examples, navigate to the `src/main/java/com/expediagroup/sdk/rapid/examples` directory and run the `RapidSdkDemoApplication.java` file. This application runs a series of scenarios demonstrating various functionalities of the RAPID SDK. Feel free to add or remove scenarios you want to run.
+
+## Dependencies
```
Comment (mohnoor94) [inline examples/README.md:28]: I don't think this is a useful info to have here.

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+This repository contains examples of how to use the Expedia Group Rapid Java SDK. The examples are written in Java and use Maven for dependency management.
+
+## Examples
+
+The example implementation provided in the examples package demonstrate different scenarios utilizing various RAPID APIs through the SDK:
+
+The scenarios below specifically demonstrate the usage of booking APIs:
+
+1.`SingleRoomBookScenario.java`: This example demonstrates the process of booking a single room. It includes steps like getting property availability, checking room prices, and finally booking a room.
```
Comment (mohnoor94) [inline examples/README.md:11]: We could add _relative_ links to all the classes mentioned in the file (as well as the "examples" project).

File: examples/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+2.`MultiRoomHoldAndResumeBookScenario.java`: This example demonstrates how to hold multiple rooms for booking and then resume the booking process. It includes steps like creating a booking with hold, resuming the booking process, and verifying the booking has been resumed properly.
+
+More scenarios are included in the examples package to demonstrate the usage of other RAPID APIs.
+
+## Setup
+
+1. Clone the repository.
+2. Navigate to the project directory `examples`.
+3. Acquire your own API key and secret from Expedia Group. Replace the placeholders `your_api_key` and `your_api_secret` in the `RapidService` class with your actual API key and secret.
```
Comment (mohnoor94) [inline examples/README.md:21]: This should be a prerequisite, we can move it above, or rewrite it to something like: `Make sure you already have your own API key....`

We could also add a link to the [Getting started with Rapid](https://developers.expediagroup.com/docs/products/rapid/setup/getting-started#getting-started-with-rapid) page.

**Review by mohnoor94 — CHANGES_REQUESTED** [review]

File: examples/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Rapid Java SDK Examples
+
+This repository contains examples of how to use the Expedia Group Rapid Java SDK. The examples are written in Java and use Maven for dependency management.
+
+## Examples
+
+The example implementation provided in the examples package demonstrate different scenarios utilizing various RAPID APIs through the SDK:
```
Comment (anssari1) [inline examples/README.md:7]: demonstrates

File: examples/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+More scenarios are included in the [examples package](src/main/java/com/expediagroup/sdk/rapid/examples/scenarios) to demonstrate the usage of other RAPID APIs.
+
+## Requirements
+- Ensure you have a valid API key and secret from Expedia Group. Check [Getting started with RAPID](https://developers.expediagroup.com/docs/products/rapid/setup/getting-started#getting-started-with-rapid|Getting) for more info.
+- Java 1.8 or higher
+- Maven
+
+## Setup
+1. Clone the repository.
+2. Navigate to the project directory `examples`.
```
Comment (mohnoor94) [inline examples/README.md:24]: Can you please add the links to examples and here and to the rest of classes in the next sections too? (RapidService, RapidSdkDemoApplication.java, .../sdk/rapid/examples)?

**Review by mohnoor94 — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 6 above)
Comment (dtayeh) [reply examples/README.md:24]: This readme is within the examples package already

**Review by mohnoor94 — APPROVED** [review]

**Review by anssari1 — APPROVED** [review]

**Pull request merged**